credential/manifest/model.go

```diff
@@ -144,8 +145,9 @@ func (cf *CredentialResponse) IsValid() error {
 }
 
 // IsValidCredentialApplicationForManifest validates the rules on how a credential manifest [cm] and credential application [ca] relate to each other https://identity.foundation/credential-manifest/#credential-application
-func IsValidCredentialApplicationForManifest(cm CredentialManifest, ca CredentialApplication) error {
+func IsValidCredentialApplicationForManifest(cm CredentialManifest, ca CredentialApplication, vcs []credential.VerifiableCredential) error {
```

Comment:
nit: consider making this a variadic argument so it can be an optional parameter 

```
IsValidCredentialApplicationForManifest(cm CredentialManifest, ca CredentialApplication, vcs ...credential.VerifiableCredential)
```

Reply:
Hmm, so if there is no vcs then would it be expected that the application's presentation_submission's descriptor_map be empty? 
<img width="537" alt="image" src="https://user-images.githubusercontent.com/5314059/196268842-3a3fa19a-ac18-420b-bb5e-b3e88fcb5bca.png">

Reply:
yes, exactly

Reply:
if there are input descriptors there must be VCs, so if there's a presentation definition at all - there must be VCs

Reply:
ok good stuff, I was trying to make a presentation definition with 0 input descriptors, 
but yup this works now with just an empty presentation definition